nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -58,6 +60,24 @@
 """
 
 
+class ColumnLinear(tensor_parallel.ColumnParallelLinear):
```

Comment:
if APEX is false, then tensor_parallel import will not happen, resulting in a tensorparallel not found kind of error right?

Reply:
APEX is now a requirement.